datasetinsights/stats/visualization/plots.py

```diff
@@ -95,7 +98,9 @@ def _process_label(bbox, label_mappings=None):
     """
     if label_mappings is not None:
         label = label_mappings[bbox.label]
-    if bbox.score != 1.0:
+    else:
+        label = None
```

Comment:
This should be `label=bbox.label` and you don't need to handle the case when `label=None` in line 103